lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -211,11 +205,26 @@ def _get_context():
         iface_name = get_service_config_value(
             'spgw', 'sgw_s5s8_up_iface_name_non_nat', '',
         )
+    return iface_name
+
+def _get_sgw_s5s8_ip_from_if_cidr(s5s8_if_name):
+    if not s5s8_if_name:
+        return ""
+    return get_ip_from_if_cidr(s5s8_if_name),
+
+def _get_context():
+    """
+    Create the context which has the interface IP and the OAI log level to use.
+    """
+    mme_service_config = load_service_mconfig('mme', MME())
+    nat = _get_enable_nat(mme_service_config)
```

Comment:
not related to your changes, but let's rename `nat` to `nat_enabled` for more meaningful naming